frontend/front/src/pages/Admin/survey/SurveyProfile.js

```diff
@@ -60,9 +60,12 @@ const SurveyProfile = () => {
     navigate(0); // reload page to update data
   }, [deleteSurveyVisit, surveyVisitId, navigate]);
 
-  const PageContent = useCallback(() => {
```

Comment:
Note to self: treating a useCallback like a proper react component causes rendering issues